vulns.go

```diff
@@ -146,20 +150,23 @@ func getSnykOpenSourceIssueWithoutTickets(flags flags, projectID string, maturit
 
 					ProjectIssuePathData, err := makeSnykAPIRequest("GET", flags.mandatoryFlags.endpointAPI+"/v1/org/"+flags.mandatoryFlags.orgID+"/project/"+projectID+"/issue/"+issueId+"/paths", flags.mandatoryFlags.apiToken, nil, customDebug)
 					if err != nil {
-						log.Printf("*** ERROR *** Could not get aggregated data from %s org %s project %s issue %s", flags.mandatoryFlags.endpointAPI, flags.mandatoryFlags.orgID, projectID, issueId)
-						log.Fatalln(err)
+						log.Printf("*** ERROR *** Could not get paths data from %s org %s project %s issue %s", flags.mandatoryFlags.endpointAPI, flags.mandatoryFlags.orgID, projectID, issueId)
+						issueSkipped += "\nissue ID: " + issueId + " from project ID:" + projectID
+						continue
 					}
 					ProjectIssuePathDataJson, er := jsn.NewJson(ProjectIssuePathData)
 					if er != nil {
 						log.Printf("*** ERROR *** Json creation failed\n")
-						log.Fatalln(er)
+						issueSkipped += "\nissue ID: " + issueId + " from project ID:" + projectID
+						continue
 					}
 					vulnsPerPath["from"] = ProjectIssuePathDataJson.K("paths")
 					marshalledvulnsPerPath, err := json.Marshal(vulnsPerPath)
 					vulnsWithAllPaths[issueId], err = jsn.NewJson(marshalledvulnsPerPath)
 					if er != nil {
 						log.Printf("*** ERROR *** Json creation failed\n")
-						log.Fatalln(er)
+						issueSkipped += "\nissue ID: " + issueId + " from project ID:" + projectID
```

Comment:
same thing here. manipulating json marshalling unmarshalling. either is works, either it fails getting the data and you won't get to that point. keep the er for json stuff in case it fails IMO.

Reply:
I removed it because then we exit and won't try the next vuln just because we can't marshalling unmarshalling that specific issue... thought we could skip it and try the next ones.